### PR TITLE
feat: add display score and background

### PR DESCRIPTION
**Pull request by AxelF44**
I've implemented the system of score using the class user, and the display of score and background !

### CODE REVIEW

File: include/Renderer.hpp [hunk 1]
```diff
@@ -51,7 +54,30 @@ class FlameClicker::Renderer {
...
+        void _renderBackground();
+
+        /**
+         * @brief Render the score.
+         *
+         */
+        void _renderScore();
+
+        /**
+         * @brief Handle all events
```
Comment (mathieurobert1) [inline include/Renderer.hpp:71]:
```suggestion
         * @brief Handle all events.
```

Comment (Njord201) [inline include/Renderer.hpp:71]:
```suggestion
         * @brief Handle all events.
```

File: include/User/User.hpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+#include <vector>
+#include <memory>
+#include <string>
+#include <iostream>
+#include <SFML/Graphics.hpp>
+
+namespace FlameClicker
+{
+    /**
+     * @brief User class of FlameClicker
```
Comment (mathieurobert1) [inline include/User/User.hpp:19]:
```suggestion
     * @brief User class of FlameClicker.
```

Comment (Njord201) [inline include/User/User.hpp:19]:
```suggestion
     * @brief User class of FlameClicker.
```

**Review by mathieurobert1 — CHANGES_REQUESTED** [review]

**Review by Njord201 — CHANGES_REQUESTED** [review]

**Review by mathieurobert1 — APPROVED** [review]

**Review by Njord201 — APPROVED** [review]

**Review by theophile-jr — APPROVED** [review]

**Review by Marius-P1 — APPROVED** [review]
:ok_hand:

**Pull request merged**